plugins/copr.py

```diff
@@ -479,6 +480,45 @@ def _download_repo(self, project_name, repo_filename, chroot=None):
         shutil.copy2(f.name, repo_filename)
         os.chmod(repo_filename, stat.S_IRUSR | stat.S_IWUSR | stat.S_IRGRP | stat.S_IROTH)
 
+    def _runtime_deps_warning(self, copr_username, copr_projectname):
+        self.base.reset(repos=True)
+        self.base.read_all_repos()
+
+        repo = self._get_copr_repo(self._sanitize_username(copr_username), copr_projectname)
+
+        runtime_deps = []
+        for repo_id in repo.cfg.sections():
+            if repo_id.startswith("copr:"):
+                continue
+            runtime_deps.append(repo_id)
```

Comment:
@dturecek thanks for extending the commit description. Do I understand correctly the main repo you are talking about is the one that starts with "copr:" and then there may be other repository configurations in the config file and those are considered the dependencies?

Might be a good idea to sum this up in a comment here (e.g. for this line, or in a docstring for this function), to help clarify how this works.

Reply:
That is correct. I've added a docstring with a short description.